### ISSUE TITLE
Bug report - Octave says "error while loading shared libraries"

### ISSUE THREAD

**Issue by apandada1** [CypherpunkArmory/UserLAnd#994]
# Describe the bug

I installed xfce in Debian and installed Octave on top of that ('sudo apt install octave')
When I run Octave, it says `/usr/lib/arm-linux-gnueabihf/octave/4.4.1/exc/arm-unknown-linux-gnueabihf/octave-gui: error while loading shared libraries: libopenblas.so.0: cannot enable executable stack as shared object requires: Permission denied `. Neither `octave-cli`, nor `octave-gui` opens.

# Steps to reproduce the behavior:

1. Install Debian distribution.

2. Install XFCE desktop.

3. Open VNC viewer, and install octave with `sudo apt install octave`.

4. It says there were errors while configuring Octave.

5. When I run Octave, I get the above-mentioned error.

# Screenshots
![xfce](https://user-images.githubusercontent.com/3063132/62820105-4ede2c00-bb7d-11e9-9420-8bd745ec9b2a.png)

# Device Information

Device: Reliance Flame 8.  
Android Version: 6.0.1.  
UserLAnd Version 2.6.1.  
The device is not rooted and not running Lineage OS.

The error about not preloading `libNimsWrap.so` was there even before installing Octave.

**Comment by apandada1** [comment]
There is some sort of bug in the `libblas.so` and `liblapack.so` provided by the [libopenblas-dev package](https://packages.ubuntu.com/bionic/armhf/libopenblas-dev/filelist). I tried to run a C program that uses BLAS, and got the same error.

I configured update-alternatives to use BLAS instead of openblas for libblas.so and liblapack.so

`sudo update-alternatives --config libblas.so.3-arm-linux-gnuabihf && sudo update-alternatives --config liblapack.so.3-arm-linux-gnuabihf`

Now Octave works and so does my C program.
